cmd/ddev/cmd/networks_test.go

```diff
@@ -17,6 +17,9 @@ func TestNetworkDuplicates(t *testing.T) {
 	assert := asrt.New(t)
 
 	ctx, client := dockerutil.GetDockerClient()
+	if dockerutil.GetDockerClientErr() != nil {
+		t.Fatalf("Could not get docker client: %v", dockerutil.GetDockerClientErr())
+	}
```

Comment:
This will give us better errors instead of possible panic when test runners suddenly fail.